config/exercise_readme.go.tmpl

```diff
@@ -17,11 +17,28 @@ Execute the tests with:
 $ cargo test
 ```
 
-All but the first test have been ignored.  After you get the first test to
-pass, remove the ignore flag (`#[ignore]`) from the next test and get the tests
-to pass again.  The test file is located in the `tests` directory.   You can
-also remove the ignore flag from all the tests to get them to run all at once
-if you wish.
+All but the first test have been ignored. If you wish to run every possible test use:
```

Comment:
Running `cargo test -- --ignored` does not always run every possible test; certain exercises have optional tests locked behind feature flags. Consider wording this as "If you wish to run all tests, use:".